src/plugins/elasticsearch/index.js

```diff
@@ -49,12 +52,28 @@ module.exports = function (kibana) {
         return reply.continue();
       }
 
+      function noCreateIndex(request, reply) {
+        const requestPath = _.trim(request.path);
+        const matchPath = createProxy.createPath(config.get('kibana.index'));
+
+        if (requestPath === matchPath) {
+          return reply(Boom.methodNotAllowed('You cannot modify the primary kibana index through this interface.'));
+        }
+
+        reply.continue();
+      }
+
+      // These routes are actually used to deal with things such as managing
+      // index patterns and advanced settings, but since hapi treats route
+      // wildcards as zero-or-more, the routes also match the kibana index
+      // itself. The client-side kibana code does not deal with creating nor
+      // destroying the kibana index, so we limit that ability here.
```

Comment:
This prevents the frontend from interacting with .kibana directly, but .kibana could still get created automatically if the frontend issued a request to `/elasticsearch/.kibana/index-pattern/` for instance. Will that be an issue at all for what you're trying to accomplish here?